lib/active_model_serializers/json_api_objects/json_api.rb

```diff
@@ -0,0 +1,26 @@
+module ActiveModelSerializers::JsonApiObjects
+  module JsonApi
```

Comment:
You mean ref the json api spec?  add just adds the jsonapi object if it's configured to add it